server/src/bots/telegram/services/messageRegistry.ts

```diff
@@ -1,24 +1,32 @@
 import {getChatId} from '../utils/chat';
 import {logger} from '../../../utils/logger';
+import {UserRegExps} from '../../../models/constants';
 
 class MessageRegistry {
     // TODO: change type to unknown and Handle casting to BotType
     _bot: any;
     _cbQueryHandlers: { [regexp: string]: CallBackQueryHandler } = {};
     _messageHandlers: { [regexp: string]: CallBackQueryHandler } = {};
 
+    _singleParameterCommandRegex: RegExp;
+
     public setBot(bot): void {
         this._bot = bot;
         this.registerCallBackQuery();
     }
 
-    public registerMessageHandler(regexp: string, callback: (bot: unknown, message: unknown, chatId: number) => unknown): MessageRegistry {
+    public addSingelParameterCommands(commands: Array<UserRegExps>){
```

Comment:
as usual I'm trying to figure out how to make our app, more configurable and keep coding less.
So I've figured out that we already have have tons of utilities doing whatever u need, instead of clear approach.
@danbilokha let's discuss it offline

Reply:
let's )